dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -222,33 +222,33 @@ ANNO_CLASSINFO_CLASS_NOTFOUND.useraction=Verify that the class is included in th
 # # {2} The error message from the exception
 ANNO_INFOSTORE_OPEN1_EXCEPTION=CWWKC0026W: [ {0} ]: The open of [{1}] caused an exception. The message is: {2}
 ANNO_INFOSTORE_OPEN1_EXCEPTION.explanation=The open of a class source caused an exception.
-ANNO_INFOSTORE_OPEN1_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_INFOSTORE_OPEN1_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} A hashcode id used to identify the object causing the exception
 ANNO_INFOSTORE_CLOSE1_EXCEPTION=CWWKC0027W: [ {0} ]: The close of [{1}] caused an exception.
 ANNO_INFOSTORE_CLOSE1_EXCEPTION.explanation=The close of a class source caused an exception.
-ANNO_INFOSTORE_CLOSE1_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_INFOSTORE_CLOSE1_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.